quarkus-mandrel-builder-image/mandrel.yaml

```diff
@@ -1,58 +1,5 @@
 images:
 
-  - graalvm-version: 22.3.2.1-Final
-    java-version: 17
-    variants:
-      - sha: 7c9f0cfc18137fd1ae976d9a4aacbadf764a6a07c7691481a8d8d9b78903212a
-        arch: amd64
-      - sha: 95995f1b99eb9323e214ea147624651366129b9cfb3e07590eeac59f02f47ed4
-        arch: arm64
-
-  - graalvm-version: 22.3.3.1-Final
-    java-version: 17
-    tag: 22.3-java17
-    variants:
-      - sha: 8449acd4426dd0b4bc278b167d2a9e729695d45ba0c75a65e6a894ce11e0aa00
-        arch: amd64
-      - sha: c8232f58868aaf6356996ee2e1958e1164e77070458849bb1890870121694245
-        arch: arm64
-
-  - graalvm-version: 23.0.0.0-Final
-    java-version: 17
-    tags: jdk-17.0.7
-    variants:
-      - sha: 3b5d4ea87f6c1cac698942068702e111ad44fd975517fb859aac11dd573f6dbb
-        arch: amd64
-      - sha: 5ee2ad025929fb4ea6f3c7c1933b04dbd9b5efd75d7c4ee7d82dd0cd47cafd9b
-        arch: arm64
-
-  - graalvm-version: 23.0.0.0-Final
-    java-version: 20
-    tags: jdk-20.0.1
-    variants:
-      - sha: 4b120a3272b4e637c3832b6245e100105ff07f46ad5b304f4844de5fad6b7fb7
-        arch: amd64
-      - sha: 3179c87dc9336a0283a5da2c3ba74ddbd4fbec52d9d8dd78cb6979b34ce8464a
-        arch: arm64
-
-  - graalvm-version: 23.0.1.2-Final
-    java-version: 17
-    tags: 23.0-java17, 23.0-jdk-17, jdk-17, jdk-17.0.8
-    variants:
-      - sha: cc73adfae094d05367e008d8ee3eebf7a08a3b45b504e55771dc739a7faabe3b
-        arch: amd64
-      - sha: 373d8341cfa678e3bdffeb8936c3edd8c353b19fdd54ba4cbe6711c6c1faa127
-        arch: arm64
```

Comment:
Please keep these around, 23.0 is an LTS release.

Reply:
Yes, my mistake.